rxn4chemistry/decorators.py

```diff
@@ -5,14 +5,14 @@
 import time
 import logging
 import threading
-from typing import Callable
+from typing import Optional, Callable
 from functools import wraps, partial
 
 from .callbacks import default_on_success
 
 LOGGER = logging.getLogger('rxn4chemistry:decorators')
-MAXIMUM_REQUESTS_PER_MINUTE = 5
-MININUM_TIMEOUT_BETWEEN_REQUESTS = 2  # expressed in seconds
+MAXIMUM_REQUESTS_PER_MINUTE = 1000
```

Comment:
since we have limits per user group here I relaxed limits, in any case the API will return an error